tools/bq-visualizer/src/app/app.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="header">
   <h1 class="header__title">BQ Visualiser</h1>
   <span class="flex-spacer"></span>
-  <a mat-button [disabled]="this.isLoggedIn" (click)="login()">login</a>
-  <a mat-button [disabled]="!this.isLoggedIn" (click)="logout()">logout</a>
+  <button mat-button [disabled]="this.isLoggedIn" (click)="login()">login</button>
+  <button mat-button [disabled]="!this.isLoggedIn" (click)="logout()">logout</button>
 
-  <a mat-button href="/terms">Terms &amp; Privacy</a>
+  <button mat-button href="/terms">Terms</button>
```

Comment:
`<button>` has no `href` attribute, so I do not believe this works.

Reply:
changed to an <a> element